### ISSUE TITLE
Failure to parse walrus/assignment operator

### ISSUE THREAD

**Issue by piacenti** [Instagram/MonkeyType#167]
I have some code that contains a code block that looks like this
```
        while parent := current_node.parent:
            result.append(current_node.name)
            current_node = parent
```
When I run `monkeytype stub` it works fine. However, when I try to `apply` I get the following error
` Cannot parse: 33:22:         while parent := current_node.parent:`
I assume this has to do with the fact I'm using the new python 3.8 walrus/assignment operator.

**Comment by carljm** [comment]
Indeed, MonkeyType currently depends on the https://github.com/ambv/retype project for applying stubs, and retype does not yet support Python 3.8 syntax.

**Comment by carljm** [comment]
This should be fixed as of MonkeyType v20.4.2, which switches from retype to LibCST for applying types.
